with-storybook/stories/4-Font.stories.js

```diff
@@ -36,4 +36,4 @@ export function font() {
 }
 
 // On-Device Register
-storiesOf('Font', module).add('Font', font);
+storiesOf('Font', module).add('Font', () => (<CustomFontComponent/>));
```

Comment:
Not sure I understand why this needs to be changed

Reply:
Nevermind, I found your comment. https://github.com/expo/examples/issues/181#issuecomment-650463810 Good work again.